### ISSUE TITLE
Accessibility assessment 

### ISSUE THREAD

**Issue by wrightmartin** [mysociety/pfi-fault-reporting#1]
Use this ticket to document our accessibility assessment work, and to record small issues. 

Larger issues should be in their own ticket with the accessibility label 

Goals

- change colours, contrast levels and fonts
- zoom in up to 300% without the text spilling off the screen
- navigate most of the website using just a keyboard
- navigate most of the website using speech recognition software
- listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)

**Comment by wrightmartin** [comment]
- [x] Alpha banner is announced as beta banner

**Comment by wrightmartin** [comment]
- [x] Index page needs an `<h2>` after the warning panel to give the correct page structure, otherwise it appears all the content belongs to the warning panel

**Comment by wrightmartin** [comment]
- [x] Confirmation page title is incorrect

**Comment by wrightmartin** [comment]
- [x] Add language attribute to html

**Comment by wrightmartin** [comment]
300% zoom level test passes

![image](https://user-images.githubusercontent.com/2292925/109000439-a1b60080-769b-11eb-8476-f322fb3b18e6.png)
